### PR TITLE
[ENH] Require source to target data in bep guidelines

### PR DESCRIPTION
**Pull request by bendhouseart**
Addresses issue #470 

As mentioned in the issue, happy to make this a hard requirement but right now author is under the impression that the BEP process needs to be amended to do so.

Feel free to commit directly and merge if the author is wrong on this or tag him for a review.

### CODE REVIEW

**Comment by bendhouseart** [comment]
Made it a requirement.

File: docs/extensions/general-guidelines.md [hunk 1]
```diff
@@ -44,57 +44,64 @@ Development on Google Docs is preferred as this is a low barrier to entry
...
-1.  Incorporate the feedback and strive for consensus.
+1. Incorporate the feedback and strive for consensus.
 
-1.  Help to merge the extension into the main specification (this will require
+1. Help to merge the extension into the main specification (this will require
     converting the proposal to Markdown and submitting a Pull Request at
     [the bids specification repository])
 
-1.  Create example datasets.
+1. Create example datasets. Complete source data (for example dicom/edf/excel) and its
```
Comment (ericearl) [inline docs/extensions/general-guidelines.md:95]:
```suggestion
1. Create example datasets. Complete source data (for example DICOM, EDF, Excel, etc.) and its
```

File: docs/extensions/general-guidelines.md [hunk 2]
```diff
@@ -44,57 +44,64 @@ Development on Google Docs is preferred as this is a low barrier to entry
...
+    corresponding target BIDS datasets, _must be_ supplied along with instructions on how
+    to transform data from source to target. When possible data from multiple vendors,
```
Comment (ericearl) [inline docs/extensions/general-guidelines.md:97]:
```suggestion
    to transform data from source to target. When possible, data from multiple vendors
```

**Review by ericearl — APPROVED** [review]
Integrate my two suggestions and I approve!

File: docs/extensions/general-guidelines.md [hunk 3]
```diff
@@ -44,57 +44,64 @@ Development on Google Docs is preferred as this is a low barrier to entry
...
-1.  Incorporate the feedback and strive for consensus.
+1. Incorporate the feedback and strive for consensus.
 
-1.  Help to merge the extension into the main specification (this will require
+1. Help to merge the extension into the main specification (this will require
     converting the proposal to Markdown and submitting a Pull Request at
     [the bids specification repository])
 
-1.  Create example datasets.
+1. Create example datasets. Complete source data (for example DICOM, EDF, Excel, etc.) and its
```
Comment (bendhouseart) [inline docs/extensions/general-guidelines.md:95]:
```suggestion
1. Create example datasets. Complete source data (for example DICOM, EDF, Excel) and its
```

**Comment by bendhouseart** [comment]
> Integrate my two suggestions and I approve!

No bad latin! Removed your etc.

**Pull request merged**